app/manage/pods/common/sms-factory.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+module.exports = [
+  '$http',
+  function($http) {
+    var urlBase = 'http://concaria-sms.herokuapp.com/api';
```

Comment:
if we're going to proxy, I'd rather just hit `/api/sms` rather than exposing this URL